### ISSUE TITLE
ALOHA notification

### ISSUE THREAD

**Issue by vzvenyach** [18F/tock#272]
Acceptance test criteria to come...but here's the user story:

> As an employee, I want tock to provide an alert to remind me to submit a leave request in ALOHA if I have put in OOO hours in the timecard, so that I can comply with leave rules and regulations.

@batemapf in thinking about this, I wonder whether we should also add an "Office Closed" category to tock to distinguish between situations where ALOHA is required and situations where it is not.

@geramirez I wonder whether the best way to handle this is to abstract some sort of post-submit behavior that is tied to the project. I'm struggling to think of another specific example at the moment, to be honest, but it seems like there may be situations when a user should get a prompt based on entering data for a specific project.

In other words, in [models.py](https://github.com/18F/tock/blob/master/tock/projects/models.py) we'd add a field to the Project class:

```
user_notification = models.CharField(max_length=200, blank=True, null=True, verbose_name="Provide an alert to a user")
```

and in (views.py)[https://github.com/18F/tock/blob/master/tock/hours/views.py], we pass the alert for any project where `user_notification is not null`.

Thoughts?

**Comment by geramirez** [comment]
@ccostino is creating editable form that pops up when the user chooses non-billable general time. Maybe we could surface a check box that asks users if they have filled out ALOHA along the same lines.

**Comment by ccostino** [comment]
Sounds like a good idea!  The implementation would be very similar to that text field in terms of having it appear or not.
